Source/Cvent.SchemaToPoco.Core/CodeToLanguage/PhpCodeGenerator.cs

```diff
@@ -114,7 +122,7 @@ private void GenerateEnum(CodeTypeDeclaration type)
             if (type.IsEnum)
             {
                 _output.WriteLine("require_once (dirname(__FILE__) .  '/../../../../../includes/Enum.php');");
-                _output.Write("class {0} extends Enum ", StringUtils.LowerFirst(type.Name));
+                _output.Write("class {0} extends Enum ", GetSafePhpName(StringUtils.LowerFirst(type.Name)));

```

Comment:
I saw this and wondered "how can we just change the type declaration and not references" and then I remembered I was looking at PHP